mono/mini/debugger-agent.c

```diff
@@ -4227,7 +4227,7 @@ ss_calculate_framecount (void *the_tls, MonoContext *ctx, gboolean force_use_ctx
 
 	if (force_use_ctx || !tls->context.valid)
 		mono_thread_state_init_from_monoctx (&tls->context, ctx);
-	compute_frame_info (tls->thread, tls);
+	compute_frame_info (tls->thread, tls, FALSE);
```

Comment:
Could you say why it's false or true for the last arg in a line comment here and elsewhere?